lib/ffi/gdal/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module FFI
   module GDAL
-    VERSION = "1.1.0"
+    VERSION = "1.2.0.pre1"
```

Comment:
Dropping support for a ruby version is a breaking change and would necessitate a bump to 2.0